packages/abstractions/tsconfig.base.json

```diff
@@ -13,6 +13,5 @@
         "declarationMap": true,
         "sourceMap": true,
         "composite": true,
```

Comment:
```suggestion
        "composite": true
```
I'm not sure how that ever worked since it's not valid JSON....

Reply:
I think it works anyhow here, but 🤷